sql/2021/mobile-web/most_used_apps.sql

```diff
@@ -0,0 +1,23 @@
+#standardSQL
+# Most used apps - mobile only
```

Comment:
Should this query (including the file) be renamed "Most used technologies"? Apps could be confusing - particularly in mobile chapter.

Reply:
Good idea. I'll change the filename but keep the usage of `app` in the sql code to keep it consistent across other chapters and with wappalyzer